src/services/email.service.js

```diff
@@ -48,7 +48,7 @@ If you did not request any password resets, then ignore this email.`;
 const sendVerificationEmail = async (to, token) => {
   const subject = 'Email Verification';
   // replace this url with the link to the email verification page of your front-end app
-  const verificationEmailUrl = `http://link-to-app/verify-email?token=${token}`;
+  const verificationEmailUrl = `http://link-to-app/v1/auth/verify-email?token=${token}`;
```

Comment:
@KeenWarrior , I think it will be front-end url. frontend will call the api accordingly. so no changes required.